traffic_monitor/health/event.go

```diff
@@ -33,6 +35,15 @@ func (t Time) MarshalJSON() ([]byte, error) {
 	return []byte(fmt.Sprintf("%d", time.Time(t).Unix())), nil
 }
 
+func (t *Time) UnmarshalJSON(data []byte) error {
+	i, err := strconv.ParseInt(string(data), 10, 64)
+	if err != nil {
+		return errors.New("health.Time (" + string(data) + ") must be a unix epoch integer: " + err.Error())
+	}
+	*t = Time(time.Unix(i, 0))
```

Comment:
Can this be a more descriptive variable name than `i`? If someone puts 2+2 together and sees that `i` stands for `int`, its purpose is still unclear.